lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -813,64 +884,15 @@ multi_update_interleave_actions(pe_action_t *first, pe_action_t *then,
             enum action_tasks task = clone_child_action(first);
             const char *first_task = task2text(task);
```

Comment:
Still not sure (referring to a comment in a previous PR) why we do this for first and not for then. There might be a good reason related to quirks of ordering.

As a side effect of this, `clone_child_action()` calls `get_complex_task()`, so I think it's impossible for us to end up with "stopped". This process would convert it to "stop". It doesn't hurt to match both in `find_instance_action()` anyway for clarity.

Reply:
Yeah, I wanted to be comprehensive in case it gets reused in another context, or the other behavior changes.

It becomes a bit clearer with future changes